python/common/org/python/types/Str.java

```diff
@@ -708,10 +708,21 @@ private long toPositiveIndex(long index) {
     }
 
     @org.python.Method(
-            __doc__ = ""
+            __doc__ = "string.isalnum() -> bool\n" +
```

Comment:
As mentioned in the other comment – the docstrings for this `isdecimal()` should match the ones in CPython.